changelogs/fragments/1744-case-insensitive-hostname-fqdn-matching.yml

```
@@ -0,0 +1,2 @@
+minor_changes:
+  - redfish_config - redfish module_utils: case insesitive search for situations where the hostname/FQDN case on iLO doesn't match variable's case (https://github.com/ansible-collections/community.general/pull/1744).
```

Is it only used in the redfish_config module, or also in other modules? If also in other modules, all modules should be mentioned here.
```suggestion
  - "redfish_config - redfish module_utils: case insensitive search for situations where the hostname/FQDN case on iLO doesn't match variable's case (https://github.com/ansible-collections/community.general/pull/1744)."
```

---

The change is in redfish_utils. But from the user-facing perspective, it only impacts the redfish_config module. Only redfish_config calls the changed set_manager_nic() method.

---

Thanks @billdodd for the input. To my knowledge the IP management is done only by redfish_config module.

---

@felixfontein I made it clearer and fixed typo in: 60c6f52a2fe3af83d4c6df62307dfe795802a618